test/NuGet.Core.Tests/NuGet.PackageManagement.Test/PackageDownloaderTests.cs

```diff
@@ -127,6 +131,49 @@ public class PackageDownloaderTests
         }
 
         [Fact]
+        public async Task TestDirectDownload_InV3()
+        {
+            // Arrange
+            var sourceRepositoryProvider = TestSourceRepositoryUtility.CreateV3OnlySourceRepositoryProvider();
+            var v3sourceRepository = sourceRepositoryProvider.GetRepositories().First();
+            var packageIdentity = new PackageIdentity("jQuery", new NuGetVersion("1.8.2"));
+
+            // Ensure package is not already cached in the Global Packages Folder
+            var globalPackagesFolder = SettingsUtility.GetGlobalPackagesFolder(Configuration.NullSettings.Instance);
```

Comment:
This test could fail if someone else installs jQuery 1.8.2 to the global packages folder while the test is running. This should be avoided by creating a test directory (`TestFileSystemUtility.CreateRandomTestFolder()`) and setting this as the global packages folder in this test. There are many tests with examples of how to use `TestFileSystemUtility`.